.github/workflows/ci.yml

```diff
@@ -49,4 +49,4 @@ jobs:
         with:
           node-version: 16
           cache: pnpm
-      - run: 'pnpm i && pnpm check && pnpm lint'
+      - run: pnpm i && pnpm check && pnpm lint && pnpm build && [ "`git status --porcelain=v1`" == "" ] # ensures that commited types are up-to-date
```

Comment:
to help contributors better understand what fails, you could split this into multiple tasks and run them even if the previous one failed like this https://github.com/sveltejs/vite-plugin-svelte/blob/main/.github/workflows/ci.yml#L69  

then looking at the failed check shows the name of the task, eg lint or check types

with this && chain its harder to see where exactly it broke

Reply:
adjusted the CI script using that of SvelteKit, should be descriptive enough